### ISSUE TITLE
EyeWitness Gets to the last IP and won't finish

### ISSUE THREAD

**Issue by turnerdb** [ChrisTruncer/EyeWitness#312]
## OS Used - ALL Information (architecture, linux flavor, etc.)
Kali

## Pastebin link to error you are encountering
No error, the program just isn't finishing once it gets to the final IP

## Expected behavior (vs. what you encountered)
Expect the report to be generated

## Any additional information

This seems to only happen on very large scans, over 4000.  Everything runs perfect until the very end and it won't finish and then not generate the report.

**Comment by ChrisTruncer** [comment]
There could be a variety of reasons, I won't be able to narrow it down based on the information given. One quick thing you can do is ctrl + c to quit the scan, then run the Recategorize.py script and pass in the .db file from your scan (in the output folder). EyeWitness will then build a report off all data in that database (likely everything except the single URL).

**Comment by turnerdb** [comment]
Oh Awesome!  Thank you!

**Comment by Freakazoidile** [comment]
I ran into this issue, I have tried to run the Recategorize.py script on the DB but it errors out and wont generate the report. It's hung on 1 URL. I tried fixing the errors in the script(s) but still had issues. I have tried importing the database into sqlite3 to attempt to remove the URL but cannot find what I would need to modify.

Is there a way to modify the DB or any of the files to remove the offending URL EW is stuck on/skip over it?
